libtrip-notification/src/test/java/com/mapbox/navigation/trip/notification/MapboxTripNotificationTest.kt

```diff
@@ -35,6 +35,8 @@ import org.junit.Assert.assertNull
 import org.junit.Before
 import org.junit.Test
 
+private const val STOP_SESSION = "Stop session"
+private const val END_NAVIGATION = "End Navigation"
```

Comment:
👀 [_Working Effectively with Unit Tests_](https://leanpub.com/wewut/read) 
> when writing tests you should prefer DAMP (Descriptive And Maintainable Procedures) 

That includes a bunch of _procedures_ as [Expect Literals](https://leanpub.com/wewut/read#leanpub-auto-expect-literals) 😅